sdk/python/feast/infra/registry/sql.py

```diff
@@ -976,7 +986,8 @@ def _apply_object(
                 if hasattr(obj_proto, "meta") and hasattr(
                     obj_proto.meta, "created_timestamp"
                 ):
-                    obj_proto.meta.created_timestamp.FromDatetime(update_datetime)
```

Comment:
This logic seems flipped?

Reply:
If `created_timestamp` is explicitly set on proto object, use object `created_timestamp`. Otherwise set it with `update_datetime`.